pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -1,4 +1,5 @@
 %link{:href=>"/shared/css/warning-banner.css", :rel=>"stylesheet"}
+%script{src: minifiable_asset_path("js/pegasusCommon.js")}
```

Comment:
Instead of calling this pegasusCommon and having to silence it, could we create two version (that end up requiring the same code), and have each of them match the pegasus haml file names where they're being used?